demo/admin/src/utils/pageTreeNodeCategoryMapping.ts

```diff
@@ -1,22 +1,22 @@
 import { GQLPageTreeNodeCategory } from "@src/graphql.generated";
+import { pageTreeCategories } from "@src/pages/pageTreeCategories";
```

Comment:
What do you think about moving this utils into the pageTreeCategories file?

Reply:
Yes we could merge that